docs/Installation-Guide.rst

```diff
@@ -569,6 +569,8 @@ On Linux HDFS version of LightGBM can be built using **CMake** and **gcc** or **
      git clone --recursive https://github.com/Microsoft/LightGBM ; cd LightGBM
      mkdir build ; cd build
      cmake -DUSE_HDFS=ON ..
+     # if you have installed HDFS to a customized location, you should specify paths to HDFS headers (hdfs.h) and library (libhdfs.so) like the following:
+     # cmake -DUSE_HDFS=ON -DHDFS_LIB_DIR=/usr/lib/hadoop-hdfs/lib -DHDFS_INCLUDE_DIR=/usr/lib/hadoop-hdfs/include/ ..
```

Comment:
Maybe ask users to provide `HDFS_LIB` directly instead of `HDFS_LIB_DIR`?
https://github.com/Microsoft/LightGBM/blob/f55e37d02670374085d523357f0032ac544b6ed2/CMakeLists.txt#L109